tools/ci_build/github/azure-pipelines/orttraining-py-packaging-pipeline-rocm.yml

```diff
@@ -130,113 +132,113 @@ stages:
           echo "##vso[task.setvariable variable=render]$render_gid"
         displayName: 'Find video and render gid to be mapped into container'
 
-      - script: |-
-          echo "video=$video"
-          echo "render=$render"
-          docker run --rm \
-            --device=/dev/kfd \
-            --device=/dev/dri \
-            --group-add $(video) \
-            --group-add $(render) \
-            --privileged \
-            --ipc=host \
-            --network=host \
-            --cap-add=SYS_PTRACE \
-            --security-opt seccomp=unconfined \
-            --volume $(Build.SourcesDirectory):/onnxruntime_src \
-            --volume $(Build.BinariesDirectory):/build \
-            --workdir /build/Release \
-            --entrypoint /bin/bash \
-            -e HIP_VISIBLE_DEVICES \
-            -e NIGHTLY_BUILD \
-            -e BUILD_BUILDNUMBER \
-            --user onnxruntimedev \
-            onnxruntimetrainingrocmbuild-rocm$(RocmVersion) \
-               /onnxruntime_src/tools/ci_build/github/pai/pai_test_launcher.sh
-        displayName: 'Run onnxruntime unit tests (in container)'
+      # - script: |-
```

Comment:
I am curious do we have better way instead of comment it out?

Reply:
will check

Reply:
So commenting it is the only way we disable it? I am wondering we can have a swtich for running those script or not.

Reply:
All these tests, w/ newly enabled ortmodule test, are covered by AMD CI, although the runtime env is slightly different. Per discussion with @baijumeswani , we don't need these tests for packaging process. But I'd like to see if @wschin has any comment on your question above.
Thanks.

Reply:
I know we need disable the tests, what I mean is whether we should have a switch to not to run them, instead of commenting the code.

Reply:
https://github.com/microsoft/onnxruntime/blob/3e6b8d159aa8d93e2ad64ba55c7bd0311fb941b9/tools/ci_build/build.py#L172-L173

Reply:
@pengwa on what condition would we run the tests or not run them?

Reply:
> https://github.com/microsoft/onnxruntime/blob/3e6b8d159aa8d93e2ad64ba55c7bd0311fb941b9/tools/ci_build/build.py#L172-L173

Thanks, we are already having --skip_tests when building whl. FYI

Reply:
quick sync with @wschin , let me remove the code to avoid confusion. I'll add comments for future ref.